src/bin/tikv-ctl.rs

```diff
@@ -928,6 +959,23 @@ fn main() {
                 ),
         )
         .subcommand(
+            SubCommand::with_name("metrics")
+                .about("print the metrics")
+                .arg(
+                Arg::with_name("tag")
+                    .short("t")
+                    .long("tag")
+                    .takes_value(true)
+                    .multiple(true)
+                    .use_delimiter(true)
+                    .require_delimiter(true)
+                    .value_delimiter(",")
+                    .default_value(METRICS_PROMETHEUS)
+                    .help(
+                        "set the metrics tag, one of prometheus/jemalloc/rocksdb_raft/rocksdb_kv, if not specified, print all",
```

Comment:
Seems like if not specified, it only prints prometheus?

Reply:
yes, as siddontang advised above